src/System.Numerics.Vectors/src/System/Numerics/Vector.cs

```diff
@@ -610,6 +610,257 @@ public unsafe Vector(T[] values, int index)
             }
         }
 
+        /// <summary>
+        /// Constructs a vector from the given span.
+        /// The span must contain at least Vector'T.Count elements.
+        /// </summary>
+        public unsafe Vector(Span<T> values)
+            : this()
```

Comment:
Nit: `this()` should be unnecessary

Reply:
I tried removing it, but then I got this error... so followed the practise in the Vector.tt file...

```
System\Numerics\Vector.cs(617,23): error CS0171: Field 'Vector<T>.register' must be fully assigned before control is returned to the caller
```